### DISCUSSION TITLE
Link this discussion board in Google Playstore

### DISCUSSION THREAD

**Discussion by ulf1** [community/community#11043] [Mobile] [answered]
It's not easy to give you feedback because you didn't link it in the Google Playstore, e.g. contacts section.

**Comment by mxie** [comment, marked as answer]
Hi @ulf1 - thank you for your feedback!

This discussion category is already linked in our Developer Contact section. It's listed as the `Website`.

Another way to get to this discussion category is through the app: `Profile` -> ⚙ for Settings -> `Share Feedback` -> `Feature Request`.

Are these options not available to you?

**Reply by ulf1** [reply]
Yes. The new features is working. Thanks
